data/Makefile

```diff
@@ -36,15 +39,26 @@ name=$(ref)
 endif
 endif
 
-# Draft genome assembly.
-draft=$(name).supernova.scaftigs
-
-# Minimum number of overlapping minimizsers
-n=50
+# Minimum number of overlapping minimizers
+ifndef minimizer_overlap
+m=90
+else ifeq ($(minimizer_overlap), stLFR)
+m=85
+else ifeq ($(minimizer_overlap), 10X)
+m=92.5
+else
+m=$(minimizer_overlap)
+endif
 
 # Minimum/Maximum barcode multiplicity
 n1=100
 n2=5000
+ifdef minimum_barcode_multiplicity
```

Comment:
Why don't we just use the full variable name? Just thinking then we wouldn't need this code section

Reply:
Im pretty sure that was the problem I had with m that I mentioned to you on slack, it doesn't assign the value properly.

Reply:
Actually you are right, in this case, it doesnt matter since you dont have to do text processing